habitat-baselines/habitat_baselines/rl/hrl/hierarchical_policy.py

```diff
@@ -88,12 +105,11 @@ def __init__(
         )
         self._high_level_policy: HighLevelPolicy = high_level_cls(
             config.hierarchical_policy.high_level_policy,
-            osp.join(
-                full_config.habitat.task.task_spec_base_path,
-                full_config.habitat.task.task_spec + ".yaml",
-            ),
+            self._pddl_problem,
```

Comment:
Should we also make a low_level_policy that contains the skills?
- hierarchical_policy: has a high_level_policy and a low_level_policy. Uses input from both to send actions to the env.
- Low_level_policy has the skills. 
- High_level_policy has the pddl domain, that it uses to determine which low-level policy to use when.

Reply:
You and I discussed this and decided we don't need this refactor. However, adding the `nn_skills` and `oracle_skills` configs hopefully help separating LL skills from the HL policy.